packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -792,6 +793,49 @@ class GoTrueClient {
     );
   }
 
+  /// Gets all the identities linked to a user.
+  Future<List<UserIdentity>> getUserIdentities() async {
+    final res = await refreshSession();
```

Comment:
I think the new `getUser()` method is more appropriate, because it doesn't create a new session, but gets the latest user updates.

Reply:
Great call!

Reply:
Updated!